src/simple_experiments/events/utils.cljs

```diff
@@ -97,4 +97,5 @@
   (get-in db [:ui :active-patient-id]))
 
 (defn title-case [s]
-  (gstring/toTitleCase (string/lower-case s)))
+  (when s
+    (gstring/toTitleCase (string/lower-case s))))
```

Comment:
A nicer way to do this, so `title-case` doesn't return `nil`, is `(gstring/toTitleCase (string/lower-case (or s "")))`

Reply:
ack, will make this change